playbook/app/pb_kits/playbook/pb_icon_circle/_icon_circle.scss

```diff
@@ -64,3 +65,10 @@ $pb_icon_circle_sizes: (
     }
   }
 }
+
+[class^=pb_icon_circle_kit_xxs] {
+  svg {
+    width: 10px !important;
+    height: 10px !important;
+  }
+}
```

Comment:
I think, if we are needing to do this override, it should be done conditionally after line 54 inside the curly braces:

```scss
@if $name == "xxs" {
  $svg_xy: $size - 6;
  svg {
    width: #{$svg_xy} !important;
    height: #{$svg_xy} !important;
  }
}
```

Reply:
Thinking about this further, you might want to try to not rely on `!important` either. Perhaps a stronger selector would do the trick:

`.pb_button_content > span > svg`

Reply:
Last one:

Even cooler thing to do would be to create a local mixin for this:

```scss
@mixin svg_size($name, $match, $adjust) {
  @if $name == $match {
    $svg_xy: $adjustment;
    & > svg {
      width: #{$svg_xy};
      height: #{$svg_xy};
    }
  }
}

// Useage
@include svg_size($name, "xxs", $size - 6px);
```

Reply:
This is really cool, thanks for the suggestion! i made these changes and just pushed it up!!